src/Microsoft.TemplateEngine.Cli/CommandParsing/INewCommandInputExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    /// <summary>
+    /// Use these extensions to get examples of dotnet new commands.
+    /// </summary>
+    internal static class INewCommandInputExtensions
+    {
+        internal static string InstallCommandExample(this INewCommandInput command, bool withVersion = false,  string packageID = "", string version = "")
+        {
+            if (string.IsNullOrWhiteSpace(packageID))
+            {
+                return withVersion ? $"dotnet {command.CommandName} -i <PACKAGE_ID>::<VERSION>" : $"dotnet {command.CommandName} -i <PACKAGE_ID>";
```

Comment:
I wonder if all this `-i` and `-u` should really be `--install` and `--uninstall` since its easier for beginner user to understand what command is supposed to do from name...

Reply:
I agree, also considering possible deprecating `-i`

Reply:
Lets do that as part of switch to commands/tab-completion